### PR TITLE
Add annotations to configure all disks in nodepools

### PR DESCRIPTION
**Pull request by paurosello**
## Goal of this PR
- Allow disk size configuration of logging volume. New default value is 15Gb.
- Allow different values for docker and containerd volume.

## Checklist
- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/controller/key/machine_deployment.go [hunk 1]
```diff
@@ -20,6 +20,27 @@ func MachineDeploymentDockerVolumeSizeGB(cr infrastructurev1alpha3.AWSMachineDep
 	return strconv.Itoa(cr.Spec.NodePool.Machine.DockerVolumeSizeGB)
 }
 
+func MachineDeploymentContainerdVolumeSizeGB(cr infrastructurev1alpha3.AWSMachineDeployment) string {
+	result, ok := cr.ObjectMeta.Annotations[annotation.AWSContainerdVolumeSize]
+	if !ok {
```
Comment (whites11) [inline service/controller/key/machine_deployment.go:25]: why don't we try to parse the value and default if it's invalid like we do for logging volume? This way we could try to create a volume that is "blah" gigs big just by setting a wrong annotation

**Comment by paurosello** [comment]
Working on test cluster

Screenshot 2023-01-16 at 15 14 33

Screenshot 2023-01-16 at 15 13 31

Screenshot 2023-01-16 at 15 15 43

File: service/controller/key/machine_deployment.go (hunk 1 above)
Comment (paurosello) [reply service/controller/key/machine_deployment.go:25]: We could try parsing anyway I guess and still return a string number... does that sound better?

Comment (paurosello) [reply service/controller/key/machine_deployment.go:28]: just did the change, can you check now?

**Review by whites11 — APPROVED** [review]
great job man

**Pull request merged**